<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4118" started-at="2020-07-30T06:59:39Z" finished-at="2020-07-30T06:59:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4118" started-at="2020-07-30T06:59:39Z" finished-at="2020-07-30T06:59:44Z">
      <class name="con.course.code.Chapter5.TimeOutTest">
        <test-method status="FAIL" signature="testError()[pri:0, instance:con.course.code.Chapter5.TimeOutTest@6ee12bac]" name="testError" duration-ms="2034" started-at="2020-07-30T06:59:40Z" finished-at="2020-07-30T06:59:42Z">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method con.course.code.Chapter5.TimeOutTest.testError() didn't finish within the time-out 2000]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method con.course.code.Chapter5.TimeOutTest.testError() didn't finish within the time-out 2000
at java.util.concurrent.FutureTask.setException(FutureTask.java:248)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testError -->
        <test-method status="PASS" signature="testSuccess()[pri:0, instance:con.course.code.Chapter5.TimeOutTest@6ee12bac]" name="testSuccess" duration-ms="2011" started-at="2020-07-30T06:59:42Z" finished-at="2020-07-30T06:59:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSuccess -->
      </class> <!-- con.course.code.Chapter5.TimeOutTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
